websecurityscanner/setup.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Google LLC
```

Comment:
This needs to use our new setup.py template.

Reply:
I will look at language and port that in.